### DISCUSSION TITLE
how to vue2.7 in vscode support props type check

### DISCUSSION THREAD

**Discussion by zhl1232** [vuejs/language-tools#1979] [Q&A]
vue3 is support, but vue2 is not 

vue3   
![image](https://user-images.githubusercontent.com/30594030/194983461-861f5c86-f534-45c5-a303-035f064de4b7.png)

vue2  
![image](https://user-images.githubusercontent.com/30594030/194983574-7e61fbde-be6a-4f65-8575-c9ae38c5217e.png)

**Comment by johnsoncodehk** [comment]
See https://github.com/johnsoncodehk/volar/issues/1850#issuecomment-1281618777
